pyseir/cdc/output_mapper.py

```diff
@@ -0,0 +1,526 @@
+import itertools
+import os
+import us
+import numpy as np
+import pandas as pd
+from collections import defaultdict
+from datetime import date, datetime, timedelta
+from enum import Enum
+from multiprocessing import Pool
+from epiweeks import Week, Year
+from string import Template
+from pyseir import OUTPUT_DIR, load_data
+from pyseir.utils import REF_DATE
+from pyseir.cdc.utils import Target, ForecastTimeUnit, ForecastUncertainty, target_column_name
+from pyseir.inference.fit_results import load_inference_result, load_mle_model
+from pyseir.ensembles.ensemble_runner import EnsembleRunner
+
+
+"""
+This class maps current pyseir model output to match cdc format.
+
+output file should have columns:
+- forecast_date: the date on which the submitted forecast data was made available in YYYY-MM-DD format
+- target: Values in the target column must be a character (string) and has format "<day_num> day ahead <target_measure>"
+          where day_num is number of days since forecast_ate to each date in forecast time range. 
+- target_end_date: last date of forecast in YYYY-MM-DD format, will always be Saturday as defined by epi weeks.
+- location: 2 digit FIPS code
+- type: "quantile" or "point"
+- quantile: quantiles of forecast target measure, with format 0.###.
+- value: value of target measure at given quantile and forecast date for given location.
+and optional: 
+- location_name: name of the location that can be useful to identify the location. 
+
+For details on formatting, check:
+https://github.com/reichlab/covid19-forecast-hub/blob/master/data-processed/README.md
+
+The output include:
+- <forecast_date>_<team>_<model>_<fips>.csv
+  File that contain the output with above columns for a specific fips.
+- <forecast_date>_<team>_<model>.csv
+  File that contain the output with above columns for all US states fips.
+- metadata-CovidActNow.txt
+  Metadata with most uptodate forecast date.
+  
+Where default value of forecast_date, team and model can be found from corresponding global variables.
+"""
+
+
+TEAM = 'CovidActNow'
+MODEL = 'SEIR_CAN'
+
+# type of target measures
+TARGETS = ['cum death', 'inc death', 'inc hosp']
```

Comment:
To be consistent with the `Target` enum:
```suggestion
TARGETS = [Target.CUM_DEATH.value, Target.INC_DEATH.value, Target.INC_HOSP.value]
```

Reply:
I prefer to keep the strings instead of the Target objects to make them more readable. These strings are transformed to Target objects in OutputMapper.__init__.